packages/auto-label/src/auto-label.ts

```diff
@@ -269,10 +279,32 @@ handler.addLabeltoRepoAndIssue = async function addLabeltoRepoAndIssue(
       }
     } else {
       await handler.addLabels(github, owner, repo, issueNumber, [githubLabel]);
+      console.log(`Label added to ${owner}/${repo} for issue ${issueNumber} is ${githubLabel}`);
       wasNotAdded = false;
     }
   }
 
+  let foundSamplesTag = undefined;
```

Comment:
Generally you don't need to assign `undefined`, cause that's what you get by default! What's the type of `foundSamplesTag` supposed to be here?